### PR TITLE
Task-56742: fix the update for the posted notes 

### PR DESCRIPTION
**Pull request by sofyenne**
ISSUE : when saving & posting a note then get back to update the old title and content are displayed (the note activity not updated)
Fix : the note activity updated

### CODE REVIEW

**Comment by ahamdi** [comment]
@sofyenne Could you explain technically what was the problem and why you added that field in the database ?

File: notes-service/src/main/java/org/exoplatform/wiki/jpa/entity/PageEntity.java [hunk 1]
```diff
@@ -111,6 +111,9 @@ public class PageEntity extends BasePageEntity {
   @Column(name = "DELETED")
   private boolean deleted;
 
+  @Column(name = "TOBEPUBLISHED")
+  private boolean toBePublished;
```
Comment (boubaker) [inline notes-service/src/main/java/org/exoplatform/wiki/jpa/entity/PageEntity.java:115]: This modification doesn't seem fixing the real issue and the description doesn't mention the real cause of the problem and how did you analysed it and proposed this fix. We 'll have to re-work this together.

**Comment by sofyenne** [comment]
> @sofyenne Could you explain technically what was the problem and why you added that field in the database ?
@ahamdi 
If the user saved and published a note and went back to modify it, the note is updated in the note space but is not updated in the activity space .I added an attribute in the table to check the status of the note (published or not) during the update and that would solve the problem !

**Comment by boubaker** [comment]
please close this since a new PR is created https://github.com/Meeds-io/notes/pull/511

**Pull request closed without merging**